Makefile

```diff
@@ -134,15 +134,35 @@ swagger-validate:
 vscode_settings: .paasta/bin/activate .tox/py37-linux
 	.paasta/bin/python paasta_tools/contrib/ide_helper.py
 
-.PHONY: generate_paasta_playground
-generate_paasta_playground: .paasta/bin/activate .tox/py37-linux
-	.tox/py37-linux/bin/python  paasta_tools/contrib/create_paasta_playground.py
-	PAASTA_SYSTEM_CONFIG_DIR=./etc_paasta_playground/ .tox/py37-linux/bin/python -m paasta_tools.generate_deployments_for_service -d ./soa_config_playground -s compute-infra-test-service -v
+etc_paasta_playground soa_config_playground: .paasta/bin/activate .tox/py37-linux
+	.tox/py37-linux/bin/python paasta_tools/contrib/create_paasta_playground.py
+
+.PHONY: generate_deployments_for_service
+generate_deployments_for_service: | soa_config_playground .tox/py37-linux
+	export KUBECONFIG=./k8s_itests/kubeconfig;\
+	export PAASTA_SYSTEM_CONFIG_DIR=./etc_paasta_playground/;\
+	export PAASTA_TEST_CLUSTER=kind-${USER}-k8s-test;\
+	.tox/py37-linux/bin/python -m paasta_tools.list_kubernetes_service_instances -d ./soa_config_playground | cut -f1 -d"." | uniq | shuf | xargs -n 1 --no-run-if-empty \
+	.tox/py37-linux/bin/python -m paasta_tools.generate_deployments_for_service -d ./soa_config_playground -v -s
```

Comment:
just curious: is this what we do internally? the `cut` seems a bit out of place (i.e., i get the feeling we'd have changed the output or added a flag to obiviate the need for this in our internal crons)

Reply:
although, we do a `cut` internally for secret syncing, so maybe we do this internall for deployments as well...

Reply:
this is what we do Internally copied/pasted here

Reply:
the cut is for getting only services since the output will have service.instance

Reply:
we should probably add a new CLI at some point so that we can do this in a less funky way internally and here :p